bot/seasons/evergreen/game.py

```diff
@@ -0,0 +1,323 @@
+        page += f"**[{data['name']}]({data['url']})**\n"
+        page += data['summary'] + "\n\n" if 'summary' in data else "\n"
+
+        # Add release date if key is in game information
+        if 'first_release_date' in data:
+            page += f"**Release Date:** {datetime.utcfromtimestamp(data['first_release_date']).date()}\n"
+
+        # Add other information
+        page += f"**Rating:** {'{0:.2f}'.format(data['total_rating']) if 'total_rating' in data else '?'}/100 "
+        page += f":star: (based on {data['total_rating_count'] if 'total_rating_count' in data else '?'})\n"
+
+        page += f"**Platforms:** "
+        page += f"{', '.join(pf['name'] for pf in data['platforms']) if 'platforms' in data else '?'}\n"
+
+        page += f"**Status:** {GameStatus(data['status']).name if 'status' in data else '?'}\n"
+
+        if 'age_ratings' in data:
+            rating = ', '.join(f"{AgeRatingCategories(age['category']).name} {AgeRatings(age['rating']).name}"
+                               for age in data['age_ratings'])
+            page += f"**Age Ratings:** {rating}\n"
+
+        if 'involved_companies' in data:
+            companies = ', '.join(co['company']['name'] for co in data['involved_companies'])
+        else:
+            companies = "?"
+        page += f"**Made by:** {companies}\n"
+
+        page += "\n"
+        page += data['storyline'] if 'storyline' in data else ''
+
+        return page, url
+
+    async def search_games(self, client: ClientSession, search: str) -> List[str]:
+        """Search game from IGDB API by string, return listing of pages."""
+        lines = []
+
+        # Define request body of IGDB API request and do request
+        body = f"""fields name, url, storyline, total_rating, total_rating_count;
+search "{search}";
+limit 50;"""
+
+        async with client.get(
+                url=BASE_URL + "games",
+                data=body,
+                headers=HEADERS) as resp:
+            data = await resp.json()
+
+        # Loop over games, format them to good format, make line and append this to total lines
+        for game in data:
+            line = ""
+
+            # Add games name and rating, also attach URL to title
+            line += f"**[{game['name']}]({game['url']})**\n"
+            line += f"""{'{0:.2f}'.format(game['total_rating'] if 'total_rating' in game.keys() else 0)}/100 :star: (by {
+            game['total_rating_count'] if 'total_rating_count' in game else '?'} users)"""
+
+            lines.append(line)
+
+        return lines
+
+    async def get_companies_list(self, client: ClientSession, limit: int, offset: int = 0) -> List[Dict[str, Any]]:
+        """Get random Game Companies from IGDB API."""
+        # Create request body, define included fields, limit and offset, do request
+        body = f"""fields name, url, start_date, logo.image_id, developed.name, published.name, description;
+limit {limit};
+offset {offset};"""
```

Comment:
Please see my other comments regarding long string construction. Is it necessary that the query contains newlines?